.github/workflows/ruby.yml

```
@@ -0,0 +1,117 @@
+
+    - name: Run rake task for the first CI node
+      run: CI_NODE_TOTAL=2 CI_NODE_INDEX=0 KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bundle exec rake knapsack:rspec
+    - name: Run rake task for the second CI node
+      run: CI_NODE_TOTAL=2 CI_NODE_INDEX=1 KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bundle exec rake knapsack:rspec
+
+    - name: Check passing arguments to rspec. Run only specs with custom_focus tag (1/2)
+      run: KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bundle exec rake "knapsack:rspec[--tag custom_focus]"
+    - name: Check passing arguments to rspec. Run only specs with custom_focus tag (2/2)
+      run: KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bin/knapsack rspec "--tag custom_focus --profile"
+
+    - name: Run specs with custom knapsack logger
+      run: CUSTOM_LOGGER=true KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bundle exec rake knapsack:rspec
+
+    - name: Run specs for custom knapsack report path
+      run: |
+        cp knapsack_rspec_report.json knapsack_custom_rspec_report.json
+        KNAPSACK_REPORT_PATH="knapsack_custom_rspec_report.json" KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bundle exec rake knapsack:rspec
+
+    - name: Run specs when spec file was removed and still exists in knapsack report json
+      run: |
+        rm spec_examples/fast/1_spec.rb
+        KNAPSACK_TEST_FILE_PATTERN="spec_examples/**{,/*/**}/*_spec.rb" bundle exec rake knapsack:rspec
+
+    - name: Run specs from multiple directories with manually specified test_dir
+      run: KNAPSACK_TEST_DIR=spec_examples KNAPSACK_TEST_FILE_PATTERN="{spec_examples,spec_engine_examples}/**{,/*/**}/*_spec.rb" bundle exec rake knapsack:rspec
+
+    # Tests for example minitest test suite
+    - name: Generate knapsack report
+      run: KNAPSACK_GENERATE_REPORT=true bundle exec rake test
+
+    - name: Run tests with enabled time offset warning
+      run: bundle exec rake test
+
+    - name: Run rake task for the first CI node
+      run: CI_NODE_TOTAL=2 CI_NODE_INDEX=0 KNAPSACK_TEST_FILE_PATTERN="test_examples/**{,/*/**}/*_test.rb" bundle exec rake knapsack:minitest
+    - name: Run rake task for the second CI node
+      run: CI_NODE_TOTAL=2 CI_NODE_INDEX=1 KNAPSACK_TEST_FILE_PATTERN="test_examples/**{,/*/**}/*_test.rb" bundle exec rake knapsack:minitest
+
+    - name: Check passing arguments to minitest. Run verbose tests
+      run: |
+        KNAPSACK_TEST_FILE_PATTERN="test_examples/**{,/*/**}/*_test.rb" bundle exec rake "knapsack:minitest[--verbose]"
+        if [ "$TRAVIS_RUBY_VERSION" == "1.9.3" ]; then KNAPSACK_TEST_FILE_PATTERN="test_examples/**{,/*/**}/*_test.rb" bin/knapsack minitest "--verbose"; fi
+        if [ "$TRAVIS_RUBY_VERSION" != "1.9.3" ]; then KNAPSACK_TEST_FILE_PATTERN="test_examples/**{,/*/**}/*_test.rb" bin/knapsack minitest "--verbose --pride"; fi
```

We can partially revert these lines (#77):
```suggestion
        KNAPSACK_TEST_FILE_PATTERN="test_examples/**{,/*/**}/*_test.rb" bin/knapsack minitest "--verbose --pride"
```
Or fully revert all of #77 in another PR.

---

> Or fully revert all of #77 in another PR.

Full reverting $77 is prepared in #113.

---

I've merged your PR https://github.com/KnapsackPro/knapsack/pull/113

Could you apply proper changes here to remove conditional for `TRAVIS_RUBY_VERSION`?